internal/repository/prisma/job_run.go

```diff
@@ -108,7 +108,7 @@ func (j *jobRunRepository) UpdateJobRun(tenantId, jobRunId string, opts *reposit
 
 func (j *jobRunRepository) GetJobRunLookupData(tenantId, jobRunId string) (*db.JobRunLookupDataModel, error) {
 	return j.client.JobRunLookupData.FindUnique(
-		db.JobRunLookupData.JobRunIDTenantID(
+		db.JobRunLookupData.JobRunIdtenantID(
```

Comment:
This is a regression in the Go client, I created https://github.com/steebchen/prisma-client-go/issues/1132

Reply:
@steebchen any sense of when this regression will be fixed? Not sure if we should wait to merge this in until it's fixed and roll back the casing changes, or just merge and go with it for now.

Reply:
fixed just now :D please upgrade to the latest version [v0.31.3](https://github.com/steebchen/prisma-client-go/releases/tag/v0.31.3) and it should work